jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-embedded/src/main/java/org/eclipse/jetty/ee10/demos/RewriteServer.java

```diff
@@ -52,6 +54,12 @@ public static void main(String[] args) throws Exception
         Server server = createServer(port);
 
         server.start();
-        server.join();
+//        server.join();
```

Comment:
remove or comment.

Isn't a join needed to stop main exiting?

Reply:
It is not needed, as the threads started are non-daemon.
However, this change slipped through, it was just for me to test things, I'll revert.